qcelemental/molparse/to_string.py

```diff
@@ -226,6 +227,29 @@ def to_dict(self) -> Dict:
             data.keywords["scf__nopen"] = molrec["molecular_multiplicity"] - 1
             data.keywords["dft__mult"] = molrec["molecular_multiplicity"]
 
+    elif dtype == "madness":
+
+        atom_format = "{elem}"
+        ghost_format = "GH"
+        # TODO handle which units valid
+        umap = {"bohr": "atomic", "angstrom": "angstrom"}
```

Comment:
what does "atomic" signify? It sounds more like "expect Cartesian geometry" than "expect Bohr geometry". If the former, fine to hard-code it and not entangle with units. Is there a way to explicitly indicate Bohr even if that's the default?

Reply:
atomic signifies atomic units.  Madness accepts atomic, a.u. , or au.
I put `{"bohr":"atomic"}` because every other qc code uses Bohr instead of atomic and because when I used "atomic":"a

Reply:
"atomic":"atomic"  `second_line = f"""units {umap.get(units.lower())}"""` wasn't working

Reply:
ok, makes sense. in umap, the lhs are pint/qcschema units and the rhs are strings the programs use to indicate that unit. so atomic:atomic wouldn't work. if madness accepts `au`, I'd do `"bohr": "au"` as being a little clearer.